docs-kits/kits/Circularity_KIT/page-adoption-view.md

```diff
@@ -111,35 +116,35 @@ Catena-X offers a decentral, sovereign, cross-organizational data exchange. For
 
 The goal of the Material Accounting Feature inside the Catena-X consortium and association is to collaborate on exchanging standardized secondary material information across the upstream & downstream value in order to develop a standardized data model. To do so, we identified potential data points along a closed loop with OEM, recycler and supplier as stakeholders as seen in the picture below. Further feature content will follow in upcoming versions of the Circularity KIT.
 
-#### Figure 3
+**Figure 3**
```

Comment:
Please clarify if your pictures are a C-X or T-X contribution.
In case of T-X contribution, please delete C-X rights in the footer of some pictures.